### ISSUE TITLE
'slides-navigation' slides all instances of superslides on page

### ISSUE THREAD

**Issue by ndimatteo** [nicinabox/superslides#50]
I have a single-page scrolling site that utilizes multiple instances of superslides to slide content for each section. Both work great, but the slides-navigation for each is envoking ALL instances of superslides, instead of **only** the one it's a part of.

I'm calling superslides as so:

``` javascript
$('.sub-pages').each(function() {
    $(this).superslides({
        slide_easing: 'easeInOutExpo',
        slide_speed: 800,
        pagination: true,
        hashchange: false,
        scrollabe: true
    });
});
```

**Comment by ndimatteo** [comment]
Unfortunately, I've since migrated to jquery cycle to use instead. Leaving open as I think it still deserves some attention.

**Comment by nicinabox** [comment]
I know that feel bro.

I'm addressing it this week with some other issues. Feel free to send a PR if you like, it's not a hard problem to fix.

**Comment by nicinabox** [comment]
Fixed in 0.6.0. Currently on master.

**Comment by ndimatteo** [comment]
Following up on this, looks like it works great! Except that the key binding still slides all instances. Would be great if it recognized the one visible on screen, or at the very least, the ability to disable keybinding. Thoughts?
